### PR TITLE
Fixes for auto-numbering of MIPs

### PR DESCRIPTION
**Pull request by vandan**
Update the template with the right placeholder string that the action will replace with the appropriate MIP number.

Also retroactively applies the new format to existing MIPs.

### CODE REVIEW

**Review by adonesky1 — APPROVED** [review]

**Comment by vandan** [comment]
Added some adjustments that intend to enable the script to accept PRs with multiple MIP files. 
If current MIP number was 10, then the script should assign and rename the following files in a PR as follows:
mip-x.md -> mip-10.md
mip-x-foo.md -> mip-11.md
mip-x-bar.md -> mip-12.md

I figured we might as well make automatic MIP number assignment flexible to allow for multiple MIPs to come through in one PR.

File: .github/.DS_Store
Comment (jiexi) [inline .github/.DS_Store:1]: these should be removed and probably added to the gitignore

File: .github/scripts/rename_and_update_mip.js [hunk 1]
```diff
@@ -6,41 +6,44 @@ const MIPsFolderPath = path.join(__dirname, "..", "..", "MIPs");
...
     return;
   }
 
-  const nextMIPNumber = getNextMIPNumber(mipFiles);
+  mipFiles.forEach(() => {
+    const nextMIPNumber = getNextMIPNumber();
 
-  const oldFilePath = path.join(MIPsFolderPath, "mip-x.md");
-  const newFilePath = path.join(MIPsFolderPath, `mip-${nextMIPNumber}.md`);
+    const oldFilePath = path.join(MIPsFolderPath, "mip-x.md");
```
Comment (jiexi) [inline .github/scripts/rename_and_update_mip.js:30]: I'm not fully understanding how this script is able to assign multiple mip file names if this loop is hardcoded to only target `mip-x.md`

Comment (vandan) [reply .github/scripts/rename_and_update_mip.js]: Thanks, I attempted to make some changes to match on files matching the pattern mip-x*.md and then apply the automatic numbering to them.

File: MIP-Template.md
Comment (jiexi) [inline MIP-Template.md:1]: ideally this template change would have come in a separate PR before this one, but not that big of a deal

**Review by jiexi — APPROVED** [review]

File: .github/scripts/rename_and_update_mip.js [hunk 2]
```diff
@@ -6,41 +6,43 @@ const MIPsFolderPath = path.join(__dirname, "..", "..", "MIPs");
...
+    const content = fs.readFileSync(oldFilePath, "utf-8");
+    const updatedContent = content.replace(/mip: x/gi, `mip: ${nextMIPNumber}`);
 
-  fs.writeFileSync(newFilePath, updatedContent);
-  fs.unlinkSync(oldFilePath);
+    fs.writeFileSync(newFilePath, updatedContent);
+    fs.unlinkSync(oldFilePath);
 
-  execSync(`git add ${oldFilePath} ${newFilePath}`);
+    execSync(`git add "${oldFilePath}" "${newFilePath}"`);
```
Comment (jiexi) [inline .github/scripts/rename_and_update_mip.js:44]: i wrapped these paths in quotes to ensure whitespace doesn't break the command

**Pull request merged**